tests/unit/frameworks/pytorch/resnets_test.py

```diff
@@ -26,15 +26,15 @@
 
 from nupic.research.frameworks.pytorch.model_utils import count_nonzero_params
 from nupic.research.frameworks.pytorch.models.resnets import (
-    LayerParams,
     ResNet,
-    cf_dict,
-    default_sparse_params,
     resnet18,
     resnet50,
     resnet101,
 )
-from nupic.research.frameworks.pytorch.sparse_layer_params import NoactLayerParams
+from nupic.research.frameworks.pytorch.sparse_layer_params import (
+    LayerParams,
+    SpareWeightsLayerParams,
```

Comment:
Shouldn't this be `SparseWeightsLayerParams`?

Reply:
Fixed

Reply:
as well as your other comments preceding.